amps/ags/rm-automation/rm-automation-community-rest-api/src/main/java/org/alfresco/rest/v0/ExportAPI.java

```diff
@@ -0,0 +1,150 @@
+
+/**
+ * Methods to make API requests using v0 API for Exporting a file
+ *
+ * @author Shubham Jain
+ * @since 3.5.0
+ */
+
+@Component
+public class ExportAPI extends BaseAPI
+{
+    /**
+     * The URI to export an item
+     */
+    private static final String EXPORT_API = "{0}rma/admin/export";
+
+    /**
+     * List to store the Node References of items to be exported
+     */
+    private static List<String> nodeRefs = new ArrayList<>();
+
+    /**
+     * Export a single Record using V0 Export API
+     *
+     * @param user               User performing the export
+     * @param password           User's Password
+     * @param expectedStatusCode Expected Response Code
+     * @param recordName         Name of the Record to be exported
+     * @return exportAPI function
+     */
+    public HttpResponse exportRecord(String user, String password, int expectedStatusCode, String recordName)
+    {
+        final String recNodeRef = getNodeRefSpacesStore() + contentService.getNodeRef(user, password, RM_SITE_ID,
+                recordName);
+
+        nodeRefs.add(recNodeRef);
+
+        return export(user, password, expectedStatusCode, nodeRefs);
+    }
+
+    /**
+     * Export an array of records using V0 Export API
+     *
+     * @param user               User performing the export
+     * @param password           User's Password
+     * @param expectedStatusCode Expected Response Code
+     * @param recordList         List of the records to be exported
+     * @return exportAPI function
+     */
+    public HttpResponse exportRecords(String user, String password, int expectedStatusCode, List<Record> recordList)
+    {
+
+        for (int i = 0; i < recordList.size(); i++)
+        {
+            String recNodeRef = getNodeRefSpacesStore() + contentService.getNodeRef(user, password, RM_SITE_ID,
+                    recordList.get(i).getName());
+
+            nodeRefs.add(recNodeRef);
+        }
+
+        return export(user, password, expectedStatusCode, nodeRefs);
+    }
+
+    /**
+     * Export a Record Folder containing records using V0 Export API
+     *
+     * @param user               User performing the export
+     * @param password           User's Password
+     * @param expectedStatusCode Expected Response Code
+     * @param folderName         Name of the Record Folder to be exported
+     * @return exportAPI function
+     */
+    public HttpResponse exportRecordFolder(String user, String password, int expectedStatusCode, String folderName)
+    {
+
+        final String recNodeRef = getNodeRefSpacesStore() + contentService.getNodeRef(user, password, RM_SITE_ID,
+                folderName);
+
+        nodeRefs.add(recNodeRef);
+
+        return export(user, password, expectedStatusCode, nodeRefs);
+    }
+
+    /**
+     * Export API function to perform Export Operation on items with given noderefs using V0 Export Rest API
+     * @param user User performing the export
```

Comment:
Please make sure you've applied the code format.

Reply:
Noted